packages/extension-ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -37,6 +38,9 @@ const CustomFooter = styled(HelperFooter)`
 function AuthManagement({ className }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const [authList, setAuthList] = useState<AuthUrls | null>(null);
+  const onAction = useContext(ActionContext);
+
+  const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
```

Comment:
If you're using this only once maybe you can put the `/account/settings` straight  here and avoid currying?